lib/manager/terraform/lockfile/index.ts

```diff
@@ -80,40 +80,46 @@ export async function updateArtifacts({
       // update all locks in the file during maintenance --> only update version in constraints
       const maintenanceUpdates = await updateAllLocks(locks);
       updates.push(...maintenanceUpdates);
-    } else if (
-      ['provider', 'required_provider'].includes(updatedDeps[0].depType)
-    ) {
-      // update only specific locks but with constrain updates
-      const dep = updatedDeps[0];
+    } else {
+      const lockUpdates = await Promise.all(
```

Comment:
This is even more worse. It's making all updates in parallel multiplied by parallel web requests.

I think we should make this in a normal loop